openapi.yaml

```diff
@@ -733,7 +733,7 @@ definitions:
       ingredientType:
         type: string
         description: the type of the ingredient
-        enum: [meat, vegetable, fruit, other]
+        enum: [cheese, sauce, meat, vegetable, fruit, other]
```

Comment:
This enum list was repeated with the parameter `filter[ingredientType]` at `/ingredients` endpoint. Can you reuse the enum there?

Reply:
It looks like this is an [openapi 3.0 feature](https://swagger.io/docs/specification/data-models/enums/) but I'm not sure there's a good way to do it here

Reply:
Take a look at this [example](https://github.com/osu-mist/parks-api/blob/cc322d795d245380bc7cc71ea6ebaf96a9a3a8c6/openapi.yaml#L264). Let me know if that still doesn't work.

Reply:
That works. My only hesitancy with that approach was having to define the anchor in a place where it doesn't feel like it would "naturally live", ie at the top of the document in the pizza filters. But you're probably right that it's worth it to not repeat the definition.